x-pack/plugins/session_view/public/components/session_view/session_view.test.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import React from 'react';
+import { sessionViewProcessEventsMock } from '../../../common/schemas/responses/session_view_process_events.mock';
+import { AppContextTestRender, createAppRootMockRenderer } from '../../../common/mock';
+import { getSessionViewProcessEvents } from './service';
+import { SessionView } from './session_view';
+
+jest.mock('./service');
+
+const getSessionViewProcessEventsMock = getSessionViewProcessEvents as jest.Mock;
```

Comment:
Doesn't `coreSetupMocks` stub out the `http` module already? do we need to mock the entire import? I think it makes these tests less reliable if we mock out too much